op-e2e/e2eutils/disputegame/helper.go

```diff
@@ -34,7 +35,7 @@ import (
 
 const alphabetGameType uint8 = 255
 const cannonGameType uint8 = 0
-const outputCannonGameType uint8 = 0 // TODO(client-pod#43): This should be a unique game type
+const outputCannonGameType uint8 = 253
```

Comment:
Switch to game type `1` under https://github.com/ethereum-optimism/client-pod/issues/43?
```suggestion
const outputCannonGameType uint8 = 253 // TODO(client-pod#43): Switch this game type to 1
```